x-pack/test/api_integration/apis/transform/delete_transforms.ts

```diff
@@ -90,7 +95,6 @@ export default ({ getService }: FtrProviderContext) => {
             transformsInfo,
           })
           .expect(403);
-        await transform.api.waitForIndicesToExist(destinationIndex);
```

Comment:
Why did you remove that check? I think it's still good to know that the dest index wasn't deleted.